tests/test_interface_maven.py

```diff
@@ -7,11 +7,18 @@
 import audeer
 
 import audbackend
+from audbackend.backend import FileSystem
```

Comment:
I guess the motivation for this import is to make the code shorter, right? Maybe we should then also apply it in the other test files, e.g. in `tests/test_backend_filesystem.py` we are still using `audbackend.backend.FileSystem`.

Reply:
I reverted this and use now everywhere `audbackend.backend.FileSystem`, ...